packages/breadboard-cli/src/commands/import.ts

```diff
@@ -73,7 +73,7 @@ export const importGraph = async (url: string, options: ImportOptions) => {
     throw new Error("Unable to generate list of boards from an API spec.");
   }
 
-  for (const api of Object.keys(boards)) {
+  for await (const api of Object.keys(boards)) {
```

Comment:
I'm a little bit confused by this, Object.keys() doesn't return a generator.

Reply:
I don't think that one was necessary in the end, after us tracking down the synchronicity issue.
```suggestion
  for (const api of Object.keys(boards)) {
```